src/Perpetuum/Services/EventServices/EPBonusEventService.cs

```diff
@@ -49,11 +54,9 @@ public override void Update(TimeSpan time)
                 return;
 
             _endingEvent = true;
-            Task.Run(() => DoClearBonus()).ContinueWith(t =>
+            Task.Run(() =>
             {
```

Comment:
Styling Comment: If you only specify a method in the Task.Run lambda you don't need the curly braces